infrastructure/controller/pkg/endpoints/components_get.go

```diff
@@ -17,7 +17,7 @@ type ClusterGet struct {
 }
 
 func (s *ClusterGet) ServeHTTP(w http.ResponseWriter, r *http.Request) {
-	components, err := configmaps.GetComponentsConfigMaps(r.Context(), s.namespace, s.clientSet, maskSecrets)
+	components, err := k8s.GetComponentsConfigMaps(r.Context(), s.namespace, s.clientSet, maskSecrets)
```

Comment:
Does this actually work yet? 🤔  When I run the image in my local cluster this endpoint returns:

```json5
{"components":{}}
```

Or perhaps there is something more than add the image that I have to do :)

Reply:
> Does this actually work yet? thinking When I run the image in my local cluster this endpoint returns:
> 
> ```js
> {"components":{}}
> ```
> 
> Or perhaps there is something more than add the image that I have to do :)

You would also need to add the ingress rules.

Reply:
This request is directly from within the cluster. If it was a networking issue the response would be a `404`. Will do more testing later